diff --git a/build/soong/service_fuzzer_bindings.go b/build/soong/service_fuzzer_bindings.go
index cb5e61fa8..94c60fe4f 100644
--- a/build/soong/service_fuzzer_bindings.go
+++ b/build/soong/service_fuzzer_bindings.go
@@ -285,6 +285,7 @@ var (
 		"display":                                                             EXCEPTION_NO_FUZZER,
 		"dnsresolver":                                                         []string{"resolv_service_fuzzer"},
 		"domain_verification":                                                 EXCEPTION_NO_FUZZER,
+		"droiduse":                                                            EXCEPTION_NO_FUZZER,
 		"color_display":                                                       EXCEPTION_NO_FUZZER,
 		"netd_listener":                                                       EXCEPTION_NO_FUZZER,
 		"network_watchlist":                                                   EXCEPTION_NO_FUZZER,
diff --git a/private/service.te b/private/service.te
index 101a9695a..698a28e96 100644
--- a/private/service.te
+++ b/private/service.te
@@ -7,6 +7,7 @@ type bg_install_control_service,     system_api_service, system_server_service,
 type binder_stats_consumer_service,  system_server_service, service_manager_type;
 type compos_service,                 service_manager_type;
 type communal_service,               app_api_service, system_server_service, service_manager_type;
+type droiduse_service,               app_api_service, system_api_service, system_server_service, service_manager_type;
 type dynamic_system_service,         system_api_service, system_server_service, service_manager_type;
 type feature_flags_service,          app_api_service, system_server_service, service_manager_type;
 type fwk_devicestate_service, system_server_service, service_manager_type;
diff --git a/private/service_contexts b/private/service_contexts
index 84304c065..9701d9588 100644
--- a/private/service_contexts
+++ b/private/service_contexts
@@ -276,6 +276,7 @@ diskstats                                 u:object_r:diskstats_service:s0
 display                                   u:object_r:display_service:s0
 dnsresolver                               u:object_r:dnsresolver_service:s0
 domain_verification                       u:object_r:domain_verification_service:s0
+droiduse                                  u:object_r:droiduse_service:s0
 color_display                             u:object_r:color_display_service:s0
 netd_listener                             u:object_r:netd_listener_service:s0
 network_watchlist                         u:object_r:network_watchlist_service:s0
